uint8 anchorId
bool isCorrected
float32 x_pos
float32 y_pos
float32 x_var
float32 y_var